routing/routing.md

```diff
@@ -0,0 +1,60 @@
+# Routing
+
+This document depicts important information about configuring routing for an existing WSO2 product deployment in a
+ container platform.
+ 
+The following will be discussed in detail in the document.
+
+* [Exposing your application to outside](#exposing-your-application-to-outside )
```

Comment:
@ThilinaManamgoda remove the extra space in the link `(#exposing-your-application-to-outside )`. Might lead to a broken link.